### ISSUE TITLE
Enhancement: Gradient Boosting Models

### ISSUE THREAD

**Issue by alisakrasnopolina** [alisakrasnopolina/project-supervised-learning#1]
В текущей версии проекта уже реализованы классические модели (KNeighborsClassifier, DecisionTreeClassifier, LogisticRegression, SVC) и ансамблевый метод RandomForestClassifier.

Чтобы проверить, можно ли дополнительно повысить качество классификации, предлагается протестировать градиентные бустинги:

- GradientBoostingClassifier,
- XGBoost,
- LightGBM (из lightgbm).
